crates/account_sdk/Cargo.toml

```diff
@@ -30,5 +30,10 @@ toml.workspace = true
 u256-literal.workspace = true
 url.workspace = true
 wasm-bindgen.workspace = true
-webauthn-authenticator-rs.workspace = true
 webauthn-rs-proto.workspace = true
+
```

Comment:
i think tokio dep has to be excluded from wasm32 target

```toml
[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
tokio.workspace = true
```

Reply:
I've fixed this, thank you.